argschema/argschema_parser.py

```diff
@@ -60,8 +59,8 @@ def __init__(self,
             logger_name, self.args.get('log_level'))
 
     @staticmethod
-    def load_schema_with_defaults(schema, args):
-        '''load_schema_with_defaults(schema, args)
+    def load_schema_with_defaults(schema, args, safedefaults=True, **kwargs):
```

Comment:
Suggestion by @fcollman to have safedefaults set to False to maintain backward compatibility.  There is already a check which ignores any instance of `argschema.schemas.DefaultSchema`, so this should fit with the spirit of this modification.